todo/handler.go

```diff
@@ -60,7 +68,10 @@ func (h *TodoHandler) UpdateTodoCompletion(c *fiber.Ctx) error {
 		return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{"error": err.Error()})
 	}
 
-	return c.SendStatus(fiber.StatusOK)
+	return c.Status(fiber.StatusOK).JSON(fiber.Map{
+		"status":  "success",
+		"message": "Todo updated successfully",
```

Comment:
message kısmını koymana gerek yok. her endpoint için message koymak hem gereksiz bir effor hemde bi tık güvenlik açığı gibi.
update endpointine istek atıyorum burda message bilgisinin ne geldiğiyle neden ilgileneyim ki sonuçta update etmesi için istek attım ne yapacağını biliyorum. o yüzden message kısmı gereksiz.

status kısmı mantıklı